### DISCUSSION TITLE
The new Beta Driver 🍻

### DISCUSSION THREAD

**Discussion by Geemahn** [NVIDIA/open-gpu-kernel-modules#643] [General]
Feels like yesterday that the open kernel modules were released and with the new Beta version 555.42.02 the open modules are the default for all GPUs which support it. Gratz to those who worked to make it happen 🍻 😁

**Comment by mtijanic** [comment]
Thanks, and cheers! Just one minor correction:

>  with the new Beta version 555.42.02 the open modules are the default for all GPUs which support it.

In 555.xx, the default is the proprietary module, but running in GSP-offload mode. This mode is functionally 99% equivalent to the open modules, but there's still a few differences. Most major one is, per [readme](https://us.download.nvidia.com/XFree86/Linux-x86_64/555.42.02/README/kernel_open.html):
> G-sync on notebooks is not yet supported by the open kernel modules.

Next major release, 560.xx, will see the open modules be the default.  That's at least the plan :)

**Reply by Geemahn** [reply]
Maybe I jumped the gun a little; I saw that I didn't need an override anymore to get the open modules to install/run and I immediately popped the champagne. Either way it is a big milestone 😄
